### ISSUE TITLE
Change to use BTC

### ISSUE THREAD

**Issue by thesoftwarejedi** [thesoftwarejedi/breadwallet#1]
Short term, just use BTC.  At some point, give option to select (open new ticket after closing this one).

See: https://github.com/voisine/breadwallet/issues/27 https://github.com/voisine/breadwallet/issues/29 https://github.com/voisine/breadwallet/issues/94

The popular, and even my personal opinion is to switch everything to bits; but the reality is that most merchants are still displaying prices in BTC. While I certainly respect @voisine hard stance on the issue of forcing users to change their mindset to bits - it's having the opposite effect on _my_ evangelizing and promoting of the bitcoin currency for use in online purchasing, in the marketplace _today_.  It is disuading many people I've shown.  

Specifically this fork was selfishly created for my 11 year old daughter B, Seth, and Anthony.
I also notice that there is other interest: @bhuddleston42

Bottom line: I've not seen a single merchant using bits.  My children shop online with their allowances - egifter, microsoft, cryfter.  This is a real world use case and I'm helping the next generation of bleeding edgers pick this up.  **Bitpay currently displays and takes payments in BTC.  So will my daughters wallet.**

**Comment by thesoftwarejedi** [comment]
pay screen is now broken, need proper decimal logic and formatting.  Will update this evening and deploy new version to test users

**Comment by voisine** [comment]
Just to make the case, I think this is a chicken and egg problem. Merchants won't change to bits until wallets do, so if wallets won't change until merchants do, it won't ever change, and bitcoin adoption will be hurt as a result. Better a little pain now to make it happen. Think of it as an investment.
